templates/field/field--multiple-values--slash.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * Field is used for multiple values field to set slash separator between rows.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+<div{{ attributes }}>{% for item in items %}{{ item.content }}{% if not loop.last %} / {% endif %}{% endfor %}</div>
```

Comment:
Let's remove this and use the existing |.

Reply:
done